### ISSUE TITLE
MR is not created for a Gitlab project

### ISSUE THREAD

**Issue by tyrcho** [fthomas/scala-steward#735]
The branch is created but not the MR. I wonder if it is because we use groups and subgroups in our gitlab organisation (on gitlab.com free plan).

Feel free to ask for more details, I'm super enthusiastic about using scala-steward for our projects and I'd gladly conduct more tests.

Here are the arguments I used: 

```
--workspace
"/data/workspace/"
--disable-sandbox
--do-not-fork
--repos-file
"/data/repos.md"
--git-author-name
"Scala Steward"
--git-author-email
"gitlab@colisweb.com"
--vcs-type
"gitlab"
--vcs-api-host
"https://gitlab.com/api/v4/"
--vcs-login
"tyrcho"
--git-ask-pass
"/data/pass.sh"
```

`/data/repos.md` contains a single line:

      - colisweb/pocs/quickstart-scala

The logs:

```
2019-07-18 15:50:42,623 INFO  Add global sbt plugins
2019-07-18 15:50:42,674 INFO  Clean workspace /data/workspace
2019-07-18 15:50:42,887 INFO  ──────────── Nurture colisweb/pocs/quickstart-scala ────────────
2019-07-18 15:50:42,888 INFO  Clone and synchronize colisweb/pocs/quickstart-scala
2019-07-18 15:50:45,826 INFO  Find updates for colisweb/pocs/quickstart-scala
2019-07-18 15:51:03,929 INFO  Ignore org.scala-lang:scala-library:provided : 2.12.6 -> 2.12.8 -> 2.13.0 (reason: ignored globally)
2019-07-18 15:51:03,929 INFO  Ignore org.scala-lang:scala-library : 2.12.8 -> 2.13.0 (reason: ignored globally)
2019-07-18 15:51:03,978 INFO  Found 4 updates:
  com.github.writethemfirst:approvals-java:test : 0.6.0 -> 0.9.0
  com.lihaoyi:pprint:test : 0.5.3 -> 0.5.5
  org.scala-sbt:sbt : 1.1.6 -> 1.2.8
  org.scalatest:scalatest:test : 3.0.5 -> 3.0.8
2019-07-18 15:51:04,025 INFO  Process update com.github.writethemfirst:approvals-java:test : 0.6.0 -> 0.9.0
2019-07-18 15:51:04,325 INFO  Trying heuristic 'strict'
2019-07-18 15:51:04,378 INFO  Create branch update/approvals-java-0.9.0
2019-07-18 15:51:07,787 INFO  Create PR update/approvals-java-0.9.0
2019-07-18 15:51:08,297 ERROR ──────────── Nurture colisweb/pocs/quickstart-scala ──────────── failed
org.http4s.client.UnexpectedStatus: unexpected HTTP status: 404 Not Found
2019-07-18 15:51:08,300 INFO  ──────────── Total time: colisweb/pocs/quickstart-scala: 25s 411ms ────────────
2019-07-18 15:51:08,301 INFO  ──────────── Total time: run: 25s 680ms ────────────
2019-07-18 15:51:08,301 INFO  Shutting down connection pool: curAllocated=0 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
```

**Comment by tyrcho** [comment]
Note : it also only create a branch for the first update found, not for the others (only `update/approvals-java-0.9.0` in my case)

**Comment by fthomas** [comment]
Maybe you get more valuable information if you turn the `ROOT_LOG_LEVEL` and `LOG_LEVEL` to `TRACE`. The [`run.sh`](https://github.com/fthomas/scala-steward/blob/master/scripts/run.sh) script shows how they can be set.

**Comment by tyrcho** [comment]
yup, here is the verbose log :

```
2019-07-18 16:22:03,415 INFO  Found 4 updates:
  com.github.writethemfirst:approvals-java:test : 0.6.0 -> 0.9.0
  com.lihaoyi:pprint:test : 0.5.3 -> 0.5.5
  org.scala-sbt:sbt : 1.1.6 -> 1.2.8
  org.scalatest:scalatest:test : 3.0.5 -> 3.0.8
2019-07-18 16:22:03,417 DEBUG Execute git rev-parse --verify master
2019-07-18 16:22:03,459 INFO  Process update com.github.writethemfirst:approvals-java:test : 0.6.0 -> 0.9.0
2019-07-18 16:22:03,465 DEBUG Requesting connection: curAllocated=1 idleQueues.size=1 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:22:03,466 DEBUG Recycling connection: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:22:03,466 DEBUG Starting up.
2019-07-18 16:22:03,466 DEBUG Starting response header timeout stage with timeout of 10000 ms
2019-07-18 16:22:03,468 DEBUG Connection was idle. Running.
2019-07-18 16:22:03,468 DEBUG Beginning request: GET https://gitlab.com/api/v4/projects/colisweb%2Fpocs%2Fquickstart-scala/merge_requests?source_branch=update/approvals-java-0.9.0
2019-07-18 16:22:03,469 TRACE Using Caching Chunk Encoder
2019-07-18 16:22:03,469 DEBUG Starting idle timeout stage with timeout of 60000 ms
2019-07-18 16:22:03,470 TRACE 429639861: continueWrite completed in 262795ns
2019-07-18 16:22:03,686 TRACE Allocating thread local ByteBuffer(16921)
2019-07-18 16:22:03,686 DEBUG SSL Read Request Status: Status = OK HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 1196 bytesProduced = 1167, java.nio.HeapByteBuffer[pos=1167 lim=16921 cap=16921]
2019-07-18 16:22:03,686 DEBUG SSL Read Request Status: Status = BUFFER_UNDERFLOW HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 0 bytesProduced = 0, java.nio.HeapByteBuffer[pos=0 lim=16921 cap=16921]
2019-07-18 16:22:03,686 TRACE 429639861: doRead completed in 527392ns
2019-07-18 16:22:03,686 DEBUG SSL Read Request Status: Status = BUFFER_UNDERFLOW HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 0 bytesProduced = 0, java.nio.HeapByteBuffer[pos=0 lim=16921 cap=16921]
2019-07-18 16:22:03,686 TRACE 429639861: doRead completed in 78376ns
2019-07-18 16:22:03,713 DEBUG Resetting org.http4s.client.blaze.Http1Connection after completing request.
2019-07-18 16:22:03,713 DEBUG Shutting down response header timeout stage
2019-07-18 16:22:03,714 DEBUG Shutting down.
2019-07-18 16:22:03,714 DEBUG Removed mid-stage: ResponseHeaderTimeoutStage
2019-07-18 16:22:03,715 DEBUG Recycling connection: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:22:03,715 DEBUG Shutting down idle timeout stage
2019-07-18 16:22:03,715 DEBUG Shutting down.
2019-07-18 16:22:03,715 DEBUG Removed mid-stage: IdleTimeoutStage
2019-07-18 16:22:03,715 DEBUG Returning idle connection to pool: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:22:03,815 INFO  Trying heuristic 'strict'
2019-07-18 16:22:03,835 DEBUG Execute git status --porcelain --untracked-files=no
2019-07-18 16:22:03,853 DEBUG Execute git rev-parse --abbrev-ref HEAD
2019-07-18 16:22:03,868 INFO  Create branch update/approvals-java-0.9.0
2019-07-18 16:22:03,869 DEBUG Execute git checkout -b update/approvals-java-0.9.0
2019-07-18 16:22:03,893 DEBUG Execute git commit --all -m Update approvals-java to 0.9.0 --no-gpg-sign
2019-07-18 16:22:03,923 DEBUG Execute git push --force --set-upstream origin update/approvals-java-0.9.0
2019-07-18 16:22:06,576 INFO  Create PR update/approvals-java-0.9.0
2019-07-18 16:22:06,597 DEBUG Requesting connection: curAllocated=1 idleQueues.size=1 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:22:06,597 DEBUG Recycling connection: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:22:06,597 DEBUG Starting up.
2019-07-18 16:22:06,597 DEBUG Starting response header timeout stage with timeout of 10000 ms
2019-07-18 16:22:06,597 DEBUG Connection was idle. Running.
2019-07-18 16:22:06,597 DEBUG Beginning request: GET https://gitlab.com/api/v4/projects/colisweb%2Fpocs%2Fquickstart-scala
2019-07-18 16:22:06,598 DEBUG Starting idle timeout stage with timeout of 60000 ms
2019-07-18 16:22:06,598 TRACE Using Caching Chunk Encoder
2019-07-18 16:22:06,599 TRACE 429639861: continueWrite completed in 246748ns
2019-07-18 16:22:06,933 TRACE Allocating thread local ByteBuffer(16921)
2019-07-18 16:22:06,934 DEBUG SSL Read Request Status: Status = OK HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 3221 bytesProduced = 3192, java.nio.HeapByteBuffer[pos=3192 lim=16921 cap=16921]
2019-07-18 16:22:06,934 DEBUG SSL Read Request Status: Status = BUFFER_UNDERFLOW HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 0 bytesProduced = 0, java.nio.HeapByteBuffer[pos=0 lim=16921 cap=16921]
2019-07-18 16:22:06,934 TRACE 429639861: doRead completed in 1253368ns
2019-07-18 16:22:06,934 DEBUG SSL Read Request Status: Status = BUFFER_UNDERFLOW HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 0 bytesProduced = 0, java.nio.HeapByteBuffer[pos=0 lim=16921 cap=16921]
2019-07-18 16:22:06,934 TRACE 429639861: doRead completed in 124302ns
2019-07-18 16:22:06,936 DEBUG Resetting org.http4s.client.blaze.Http1Connection after completing request.
2019-07-18 16:22:06,936 DEBUG Shutting down response header timeout stage
2019-07-18 16:22:06,936 DEBUG Shutting down.
2019-07-18 16:22:06,936 DEBUG Removed mid-stage: ResponseHeaderTimeoutStage
2019-07-18 16:22:06,940 DEBUG Recycling connection: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:22:06,940 DEBUG Shutting down idle timeout stage
2019-07-18 16:22:06,940 DEBUG Shutting down.
2019-07-18 16:22:06,940 DEBUG Removed mid-stage: IdleTimeoutStage
2019-07-18 16:22:06,940 DEBUG Returning idle connection to pool: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:22:06,955 DEBUG Requesting connection: curAllocated=1 idleQueues.size=1 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:22:06,955 DEBUG Recycling connection: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:22:06,955 DEBUG Starting up.
2019-07-18 16:22:06,955 DEBUG Starting response header timeout stage with timeout of 10000 ms
2019-07-18 16:22:06,955 DEBUG Connection was idle. Running.
2019-07-18 16:22:06,956 DEBUG Beginning request: POST https://gitlab.com/api/v4/projects/tyrcho%2Fquickstart-scala/merge_requests
2019-07-18 16:22:06,956 DEBUG Starting idle timeout stage with timeout of 60000 ms
2019-07-18 16:22:06,957 TRACE Using static encoder
2019-07-18 16:22:06,959 TRACE Allocating thread local ByteBuffer(16921)
2019-07-18 16:22:06,959 TRACE 429639861: continueWrite completed in 819530ns
2019-07-18 16:22:06,960 TRACE Allocating thread local ByteBuffer(16921)
2019-07-18 16:22:06,962 TRACE 429639861: continueWrite completed in 1737300ns
2019-07-18 16:22:07,166 TRACE Allocating thread local ByteBuffer(16921)
2019-07-18 16:22:07,166 DEBUG SSL Read Request Status: Status = OK HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 495 bytesProduced = 466, java.nio.HeapByteBuffer[pos=466 lim=16921 cap=16921]
2019-07-18 16:22:07,166 DEBUG SSL Read Request Status: Status = BUFFER_UNDERFLOW HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 0 bytesProduced = 0, java.nio.HeapByteBuffer[pos=0 lim=16921 cap=16921]
2019-07-18 16:22:07,167 TRACE 429639861: doRead completed in 575956ns
2019-07-18 16:22:07,167 DEBUG SSL Read Request Status: Status = BUFFER_UNDERFLOW HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 0 bytesProduced = 0, java.nio.HeapByteBuffer[pos=0 lim=16921 cap=16921]
2019-07-18 16:22:07,167 TRACE 429639861: doRead completed in 124521ns
2019-07-18 16:22:07,169 DEBUG Resetting org.http4s.client.blaze.Http1Connection after completing request.
2019-07-18 16:22:07,169 DEBUG Shutting down response header timeout stage
2019-07-18 16:22:07,169 DEBUG Shutting down.
2019-07-18 16:22:07,169 DEBUG Removed mid-stage: ResponseHeaderTimeoutStage
2019-07-18 16:22:07,172 DEBUG Shutting down idle timeout stage
2019-07-18 16:22:07,172 DEBUG Shutting down.
2019-07-18 16:22:07,172 DEBUG Removed mid-stage: IdleTimeoutStage
2019-07-18 16:22:07,172 DEBUG Shutting down.
2019-07-18 16:22:07,174 DEBUG Error while closing SSL Engine
javax.net.ssl.SSLException: Inbound closed before receiving peer's close_notify: possible truncation attack?
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:208)
	at sun.security.ssl.SSLEngineImpl.fatal(SSLEngineImpl.java:1647)
	at sun.security.ssl.SSLEngineImpl.fatal(SSLEngineImpl.java:1615)
	at sun.security.ssl.SSLEngineImpl.closeInbound(SSLEngineImpl.java:1542)
	at org.http4s.blaze.pipeline.stages.SSLStage.stageShutdown(SSLStage.scala:61)
	at org.http4s.blaze.pipeline.Tail.closePipeline(Stages.scala:78)
	at org.http4s.blaze.pipeline.Tail.closePipeline$(Stages.scala:77)
	at org.http4s.blaze.pipeline.stages.SSLStage.closePipeline(SSLStage.scala:41)
	at org.http4s.blaze.pipeline.Tail.closePipeline(Stages.scala:81)
	at org.http4s.blaze.pipeline.Tail.closePipeline$(Stages.scala:77)
	at org.http4s.client.blaze.ReadBufferStage.closePipeline(ReadBufferStage.scala:13)
	at org.http4s.blaze.pipeline.Tail.closePipeline(Stages.scala:81)
	at org.http4s.blaze.pipeline.Tail.closePipeline$(Stages.scala:77)
	at org.http4s.client.blaze.Http1Connection.closePipeline(Http1Connection.scala:25)
	at org.http4s.client.blaze.Http1Connection.shutdownWithError(Http1Connection.scala:83)
	at org.http4s.client.blaze.Http1Connection.stageShutdown(Http1Connection.scala:54)
	at org.http4s.client.blaze.Http1Connection.shutdown(Http1Connection.scala:52)
	at org.http4s.client.PoolManager.$anonfun$invalidate$1(PoolManager.scala:313)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at cats.effect.internals.IORunLoop$.cats$effect$internals$IORunLoop$$loop(IORunLoop.scala:87)
	at cats.effect.internals.IORunLoop$.startCancelable(IORunLoop.scala:41)
	at cats.effect.internals.IOBracket$BracketStart.run(IOBracket.scala:86)
	at cats.effect.internals.Trampoline.cats$effect$internals$Trampoline$$immediateLoop(Trampoline.scala:70)
	at cats.effect.internals.Trampoline.startLoop(Trampoline.scala:36)
	at cats.effect.internals.TrampolineEC$JVMTrampoline.super$startLoop(TrampolineEC.scala:93)
	at cats.effect.internals.TrampolineEC$JVMTrampoline.$anonfun$startLoop$1(TrampolineEC.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at cats.effect.internals.TrampolineEC$JVMTrampoline.startLoop(TrampolineEC.scala:93)
	at cats.effect.internals.Trampoline.execute(Trampoline.scala:43)
	at cats.effect.internals.TrampolineEC.execute(TrampolineEC.scala:44)
	at cats.effect.internals.Callback$AsyncIdempotentCallback.apply(Callback.scala:133)
	at cats.effect.internals.Callback$AsyncIdempotentCallback.apply(Callback.scala:120)
	at org.http4s.client.blaze.Http1Connection.parsePrelude(Http1Connection.scala:296)
	at org.http4s.client.blaze.Http1Connection.$anonfun$readAndParsePrelude$1(Http1Connection.scala:189)
	at org.http4s.client.blaze.Http1Connection.$anonfun$readAndParsePrelude$1$adapted(Http1Connection.scala:188)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2019-07-18 16:22:07,175 DEBUG Shutting down.
2019-07-18 16:22:07,175 DEBUG doClosePipeline(None)
2019-07-18 16:22:07,175 DEBUG Shutting down.
2019-07-18 16:22:07,176 DEBUG Invalidated connection, no pending requests. Shrinking pool: curAllocated=0 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:22:07,177 DEBUG Execute git checkout master
2019-07-18 16:22:07,196 ERROR ──────────── Nurture colisweb/pocs/quickstart-scala ──────────── failed
org.http4s.client.UnexpectedStatus: unexpected HTTP status: 404 Not Found
2019-07-18 16:22:07,198 INFO  ──────────── Total time: colisweb/pocs/quickstart-scala: 22s 13ms ────────────
2019-07-18 16:22:07,199 INFO  ──────────── Total time: run: 22s 124ms ────────────
2019-07-18 16:22:07,199 INFO  Shutting down connection pool: curAllocated=0 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false

Process finished with exit code 0
```

**Comment by tyrcho** [comment]
this POST requests looks wrong to me: `POST https://gitlab.com/api/v4/projects/tyrcho%2Fquickstart-scala/merge_requests
`

**Comment by tyrcho** [comment]
I think it should use the same URL as this one : `GET https://gitlab.com/api/v4/projects/colisweb%2Fpocs%2Fquickstart-scala`

**Comment by tyrcho** [comment]
if I change https://github.com/fthomas/scala-steward/blob/master/modules/core/src/main/scala/org/scalasteward/core/gitlab/http4s/Http4sGitlabApiAlg.scala#L126 and replace the line with `    val userOwnedRepo = repo//.copy(owner = user.login)`
The MR is created properly but there are still errors in the log and the other updates are not handled:

```
2019-07-18 16:34:39,134 DEBUG Beginning request: GET https://gitlab.com/api/v4/projects/colisweb%2Fpocs%2Fquickstart-scala
2019-07-18 16:34:39,135 DEBUG Starting idle timeout stage with timeout of 60000 ms
2019-07-18 16:34:39,624 DEBUG SSL Read Request Status: Status = OK HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 3222 bytesProduced = 3193, java.nio.HeapByteBuffer[pos=3193 lim=16921 cap=16921]
2019-07-18 16:34:39,624 DEBUG SSL Read Request Status: Status = BUFFER_UNDERFLOW HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 0 bytesProduced = 0, java.nio.HeapByteBuffer[pos=0 lim=16921 cap=16921]
2019-07-18 16:34:39,625 DEBUG SSL Read Request Status: Status = BUFFER_UNDERFLOW HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 0 bytesProduced = 0, java.nio.HeapByteBuffer[pos=0 lim=16921 cap=16921]
2019-07-18 16:34:39,626 DEBUG Resetting org.http4s.client.blaze.Http1Connection after completing request.
2019-07-18 16:34:39,626 DEBUG Shutting down response header timeout stage
2019-07-18 16:34:39,626 DEBUG Shutting down.
2019-07-18 16:34:39,626 DEBUG Removed mid-stage: ResponseHeaderTimeoutStage
2019-07-18 16:34:39,630 DEBUG Recycling connection: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:34:39,630 DEBUG Shutting down idle timeout stage
2019-07-18 16:34:39,630 DEBUG Shutting down.
2019-07-18 16:34:39,630 DEBUG Removed mid-stage: IdleTimeoutStage
2019-07-18 16:34:39,630 DEBUG Returning idle connection to pool: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:34:39,643 DEBUG Requesting connection: curAllocated=1 idleQueues.size=1 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:34:39,643 DEBUG Recycling connection: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:34:39,643 DEBUG Starting up.
2019-07-18 16:34:39,643 DEBUG Starting response header timeout stage with timeout of 10000 ms
2019-07-18 16:34:39,644 DEBUG Connection was idle. Running.
2019-07-18 16:34:39,644 DEBUG Beginning request: POST https://gitlab.com/api/v4/projects/colisweb%2Fpocs%2Fquickstart-scala/merge_requests
2019-07-18 16:34:39,644 DEBUG Starting idle timeout stage with timeout of 60000 ms
2019-07-18 16:34:40,406 DEBUG SSL Read Request Status: Status = OK HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 4040 bytesProduced = 4011, java.nio.HeapByteBuffer[pos=4011 lim=16921 cap=16921]
2019-07-18 16:34:40,406 DEBUG SSL Read Request Status: Status = BUFFER_UNDERFLOW HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 0 bytesProduced = 0, java.nio.HeapByteBuffer[pos=0 lim=16921 cap=16921]
2019-07-18 16:34:40,406 DEBUG SSL Read Request Status: Status = BUFFER_UNDERFLOW HandshakeStatus = NOT_HANDSHAKING
bytesConsumed = 0 bytesProduced = 0, java.nio.HeapByteBuffer[pos=0 lim=16921 cap=16921]
2019-07-18 16:34:40,408 DEBUG Resetting org.http4s.client.blaze.Http1Connection after completing request.
2019-07-18 16:34:40,408 DEBUG Shutting down response header timeout stage
2019-07-18 16:34:40,408 DEBUG Shutting down.
2019-07-18 16:34:40,408 DEBUG Removed mid-stage: ResponseHeaderTimeoutStage
2019-07-18 16:34:40,415 DEBUG Recycling connection: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:34:40,416 DEBUG Shutting down idle timeout stage
2019-07-18 16:34:40,416 DEBUG Shutting down.
2019-07-18 16:34:40,416 DEBUG Removed mid-stage: IdleTimeoutStage
2019-07-18 16:34:40,416 DEBUG Returning idle connection to pool: curAllocated=1 idleQueues.size=0 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:34:40,449 DEBUG Execute git checkout master
2019-07-18 16:34:40,469 ERROR ──────────── Nurture colisweb/pocs/quickstart-scala ──────────── failed
io.circe.DecodingFailure$$anon$2: [K, V]Map[K, V]: DownField(colisweb/back/talking_ant),DownField(store)
2019-07-18 16:34:40,471 INFO  ──────────── Total time: colisweb/pocs/quickstart-scala: 21s 192ms ────────────
2019-07-18 16:34:40,472 INFO  ──────────── Total time: run: 21s 283ms ────────────
2019-07-18 16:34:40,472 INFO  Shutting down connection pool: curAllocated=1 idleQueues.size=1 waitQueue.size=0 maxWaitQueueLimit=256 closed=false
2019-07-18 16:34:40,473 DEBUG Shutting down.
2019-07-18 16:34:40,474 DEBUG Error while closing SSL Engine
javax.net.ssl.SSLException: Inbound closed before receiving peer's close_notify: possible truncation attack?
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:208)
	at sun.security.ssl.SSLEngineImpl.fatal(SSLEngineImpl.java:1647)
	at sun.security.ssl.SSLEngineImpl.fatal(SSLEngineImpl.java:1615)
	at sun.security.ssl.SSLEngineImpl.closeInbound(SSLEngineImpl.java:1542)
	at org.http4s.blaze.pipeline.stages.SSLStage.stageShutdown(SSLStage.scala:61)
	at org.http4s.blaze.pipeline.Tail.closePipeline(Stages.scala:78)
	at org.http4s.blaze.pipeline.Tail.closePipeline$(Stages.scala:77)
	at org.http4s.blaze.pipeline.stages.SSLStage.closePipeline(SSLStage.scala:41)
	at org.http4s.blaze.pipeline.Tail.closePipeline(Stages.scala:81)
	at org.http4s.blaze.pipeline.Tail.closePipeline$(Stages.scala:77)
	at org.http4s.client.blaze.ReadBufferStage.closePipeline(ReadBufferStage.scala:13)
	at org.http4s.blaze.pipeline.Tail.closePipeline(Stages.scala:81)
	at org.http4s.blaze.pipeline.Tail.closePipeline$(Stages.scala:77)
	at org.http4s.client.blaze.Http1Connection.closePipeline(Http1Connection.scala:25)
	at org.http4s.client.blaze.Http1Connection.shutdownWithError(Http1Connection.scala:83)
	at org.http4s.client.blaze.Http1Connection.stageShutdown(Http1Connection.scala:54)
	at org.http4s.client.blaze.Http1Connection.shutdown(Http1Connection.scala:52)
	at org.http4s.client.PoolManager.$anonfun$shutdown$3(PoolManager.scala:357)
	at org.http4s.client.PoolManager.$anonfun$shutdown$3$adapted(PoolManager.scala:357)
	at scala.collection.LinearSeqOptimized.foreach(LinearSeqOptimized.scala:75)
	at scala.collection.LinearSeqOptimized.foreach$(LinearSeqOptimized.scala:72)
	at scala.collection.mutable.MutableList.foreach(MutableList.scala:33)
	at org.http4s.client.PoolManager.$anonfun$shutdown$2(PoolManager.scala:357)
	at org.http4s.client.PoolManager.$anonfun$shutdown$2$adapted(PoolManager.scala:357)
	at scala.collection.mutable.HashMap.$anonfun$foreach$1(HashMap.scala:149)
	at scala.collection.mutable.HashTable.foreachEntry(HashTable.scala:237)
	at scala.collection.mutable.HashTable.foreachEntry$(HashTable.scala:230)
	at scala.collection.mutable.HashMap.foreachEntry(HashMap.scala:44)
	at scala.collection.mutable.HashMap.foreach(HashMap.scala:149)
	at org.http4s.client.PoolManager.$anonfun$shutdown$1(PoolManager.scala:357)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at cats.effect.internals.IORunLoop$.cats$effect$internals$IORunLoop$$loop(IORunLoop.scala:87)
	at cats.effect.internals.IORunLoop$.startCancelable(IORunLoop.scala:41)
	at cats.effect.internals.IOBracket$BracketStart.run(IOBracket.scala:86)
	at cats.effect.internals.Trampoline.cats$effect$internals$Trampoline$$immediateLoop(Trampoline.scala:70)
	at cats.effect.internals.Trampoline.startLoop(Trampoline.scala:36)
	at cats.effect.internals.TrampolineEC$JVMTrampoline.super$startLoop(TrampolineEC.scala:93)
	at cats.effect.internals.TrampolineEC$JVMTrampoline.$anonfun$startLoop$1(TrampolineEC.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at cats.effect.internals.TrampolineEC$JVMTrampoline.startLoop(TrampolineEC.scala:93)
	at cats.effect.internals.Trampoline.execute(Trampoline.scala:43)
	at cats.effect.internals.TrampolineEC.execute(TrampolineEC.scala:44)
	at cats.effect.internals.Callback$AsyncIdempotentCallback.apply(Callback.scala:133)
	at cats.effect.internals.Callback$AsyncIdempotentCallback.apply(Callback.scala:120)
	at org.http4s.client.blaze.Http1Connection.parsePrelude(Http1Connection.scala:296)
	at org.http4s.client.blaze.Http1Connection.$anonfun$readAndParsePrelude$1(Http1Connection.scala:189)
	at org.http4s.client.blaze.Http1Connection.$anonfun$readAndParsePrelude$1$adapted(Http1Connection.scala:188)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2019-07-18 16:34:40,474 DEBUG Shutting down.
2019-07-18 16:34:40,474 DEBUG doClosePipeline(None)
2019-07-18 16:34:40,475 DEBUG Shutting down.

Process finished with exit code 0
```

**Comment by fthomas** [comment]
I think this has been fixed by #722, #738, and #742.

**Comment by tyrcho** [comment]
I confirm that this is fixed, thanks, this was super fast :)
